### PR TITLE
Clarification of audio format details of AudioPassThru

### PR DESCRIPTION
**Pull request by shoamano83**
The aim of this proposal is to clarify the details of audio data format of AudioPassThru recording.

### CODE REVIEW

**Comment by shoamano83** [comment]
Hi @theresalech, this proposal is ready for review.
Thanks.

File: proposals/nnnn-apt-clarification.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+
+## Potential downsides
+
+The author does not come up with any potential downsides.
+
+
+## Impact on existing code
+
+- The changes in interface files do not impact on any implementation.
```
Comment (theresalech) [inline proposals/nnnn-apt-clarification.md]: Please update to "The changes in interface files do not have an impact on any implementation."

**Comment by shoamano83** [comment]
Thanks @theresalech , I updated the document. Please review again.

**Comment by AndrewRMitchell** [comment]
Just one change for clarification, can we mention the fact that Audio is pegged to Channel 1 somewhere in addition to the Alternatives Considered?

i.e. in the Detailed Design can we add something like the following section:

**Audio Channel**

Since the audio format is monaural, it is necessary to pick a channel to default to. This is due to the fact that monaural audio requires a single audio source to avoid requiring mixdown, combined with the fact that most vehicles currently have only a monaural mic. For compatibility with the largest installed base, the channel selection is currently being fixed to channel 1.

**Comment by theresalech** [comment]
@AndrewRMitchell a review issue has been created for this proposal, could you please leave your comments [here](https://github.com/smartdevicelink/sdl_evolution/issues/394)?

**Pull request merged**